docs/docs/stories.mdx

```diff
@@ -18,23 +18,23 @@ Here's an example of a dialogue in the Rasa story format:
 stories:
 - story: collect restaurant booking info  # name of the story - just for debugging
   steps:
-  - intent: greet                         # user message with no entities
-  - action: utter_ask_howcanhelp
-  - intent: inform                        # user message with no entities
+  - intent: greet                          # user message with no entities
+  - action: utter_ask_howcanhelp           # action that the bot should execute
+  - intent: inform                         # user message with entities
     entities:
     - location: "rome"
     - price: "cheap"
-  - action: utter_on_it                  # action that the bot should execute
+  - bot: On it                             # actual text that bot can output
   - action: utter_ask_cuisine
-  - intent: inform
-    entities:
-    - cuisine: "spanish"
+  - user: I would like [spanish](cuisine). # actual text that user input
```

Comment:
@erohmensing What do you think of adding an explanation why we need to annotate entities like this in stories?

Reply:
Yeah that would be helpful -- as you don't have to annotate the intent, i could see the difference there being confusing

Reply:
> why we need

what do you mean by `need`, the need comes from users

Reply:
we need it as we use it to train NLU with it, no?

Reply:
we don't train NLU with it

Reply:
I wrote in the docs `In addition, you can add entity tags that will be extracted by the end-to-end policy.` I didn't write it'll be used in NLU

Reply:
rephrased it to "In addition, you can add entity tags that can be extracted by the `TEDPolicy`." so that it is explicit